freemocap/fmc_good_frame_finder.py

```diff
@@ -0,0 +1,103 @@
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+
+
+def find_velocity_values_within_limit(skeleton_velocity_data, velocity_limit):
```

Comment:
terminology-wise, this would better labeled as a "threshold" rather than a "limit."  

not a big deal, just pointing out